invoice_sale_pricelist/models/account_invoice_line.py

```diff
@@ -45,23 +37,11 @@ def _get_sale_price_unit(self):
         )
 
         return self.env["account.tax"]._fix_tax_included_price(
-            product.price, taxes, self.invoice_line_tax_ids
+            product.price, taxes, self.tax_ids
```

Comment:
The scope of _get_computed_price_unit is just to get the price unit, taxes are handled elsewhere so I think we can get rid of tax related code.